presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestPrestoDriverUri.java

```diff
@@ -256,6 +257,18 @@ public void testUriWithExtraCredentials()
         assertEquals(properties.getProperty(EXTRA_CREDENTIALS.getKey()), extraCredentials);
     }
 
+    @Test
+    public void testUriWithCustomHeaders()
+            throws SQLException, UnsupportedEncodingException
+    {
+        String extraCredentials = "{'testHeaderKey':'testHeaderValue'}";
+        String encodedExtraCredentials = URLEncoder.encode(extraCredentials, StandardCharsets.UTF_8.toString());
+        System.out.println("presto://localhost:8080?customHeaders=" + encodedExtraCredentials);
+        PrestoDriverUri parameters = createDriverUri("presto://localhost:8080?customHeaders=" + encodedExtraCredentials);
+        Properties properties = parameters.getProperties();
+        assertEquals(properties.getProperty(CUSTOM_HEADERS.getKey()), extraCredentials);
```

Comment:
extraCredentials has a diff meaning. Please rename this method block to use custom headers as the naming

Reply:
✅